subprojects/plugins/src/main/java/org/gradle/api/plugins/WarPlugin.java

```diff
@@ -46,7 +48,23 @@
  */
 public abstract class WarPlugin implements Plugin<Project> {
     public static final String PROVIDED_COMPILE_CONFIGURATION_NAME = "providedCompile";
+    /**
+     * This is the resolvable configuration that should be used to resolve the classpath containing the dependencies
+     * added to the {@link #PROVIDED_COMPILE_CONFIGURATION_NAME} configuration.
+     *
+     * @since 8.0
+     */
+    @Incubating
+    public static final String PROVIDED_COMPILE_CLASSPATH_CONFIGURATION_NAME = "providedCompileClasspath";
```

Comment:
I question whether these resolvable configurations need to be public at all. Do we expect users to be resolving these configurations themselves? I don't think so. The reason the original `providedCompile`/`providedRuntime` existed publicly were for declaring dependencies against them. I don't think we should be exposing these resolvable configurations to users. (Obviously if the user knew the name of the configuration they'd be able to do whatever they want to it, but I don't think we should make the name of this configuration non-internal). Thoughts?

Reply:
Good idea.  The configurations are already non-visible, the names seem like they should be too.  I'll update this.